analyzer/src/main/kotlin/managers/Stack.kt

```diff
@@ -347,8 +347,11 @@ class Stack(
 
         return Package(
             id = id,
-            // TODO: Find a way to track authors.
-            authors = sortedSetOf(),
+            authors = map["author"]
+                ?.split(",")
+                ?.map(String::trim)
+                ?.filter(String::isNotBlank)
```

Comment:
Nit: After trimming, strings cannot be blank anymore, so this could be just `isNotEmpty`.